test/labeling/apply/test_spark.py

```diff
@@ -53,6 +59,19 @@ def test_lf_applier_spark(self) -> None:
         L = applier.apply(rdd)
         np.testing.assert_equal(L, L_EXPECTED)
 
+    @pytest.mark.complex
+    @pytest.mark.spark
+    def test_lf_applier_spark_fault(self) -> None:
+        sc = SparkContext.getOrCreate()
+        sql = SQLContext(sc)
+        df = pd.DataFrame(dict(num=DATA))
+        rdd = sql.createDataFrame(df).rdd
+        applier = SparkLFApplier([f, f_bad])
+        with self.assertRaises(Exception):
```

Comment:
can we be more specific about the exception?

Reply:
tried but there's a race condition for which exception gets thrown first